src/sentry/api/event_search.py

```diff
@@ -119,22 +119,25 @@
 # standard key:val filter
 text_filter = negation? text_key sep operator? search_value
 
-key              = ~r"[a-zA-Z0-9_.-]+"
-quoted_key       = '"' ~r"[a-zA-Z0-9_.:-]+" '"'
-explicit_tag_key = "tags" open_bracket search_key closed_bracket
-aggregate_key    = key open_paren spaces function_args? spaces closed_paren
-function_args    = key (spaces comma spaces key)*
-search_key       = key / quoted_key
-text_key         = explicit_tag_key / search_key
-value            = ~r"[^()\t\n ]*"
-quoted_value     = '"' ('\\"' / ~r'[^"]')* '"'
-in_value         = (&in_value_termination in_value_char)+
-text_in_value    = quoted_value / in_value
-search_value     = quoted_value / value
-numeric_value    = "-"? numeric ~r"[kmb]"? &(end_value / comma / closed_bracket)
-boolean_value    = ~r"(true|1|false|0)"i &end_value
-text_in_list     = open_bracket text_in_value (spaces comma spaces text_in_value)* closed_bracket &end_value
-numeric_in_list  = open_bracket numeric_value (spaces comma spaces numeric_value)* closed_bracket &end_value
+key                    = ~r"[a-zA-Z0-9_.@-]+"
+quoted_key             = '"' ~r"[a-zA-Z0-9_.@:-]+" '"'
```

Comment:
Why do `key` and `quoted_key` need `@` in the pattern? Does it not suffice to allow it in `aggregate_param`?

Reply:
hm, might be leftover as I was working on this, should be safe to remove :+1: